project4/project/ebs/templates/ebs/home.html

```diff
@@ -322,116 +320,162 @@ <h4 class="fontfamily1">Fashion Week Parties: Night 4</h4></a>
 				<div class="col-md-4 ">
```

Comment:
HTML5 semantic markup not used. use section, main, aside and other semantic markup while using HTML5